addons/crm/static/src/js/tour.js

```diff
@@ -8,8 +8,8 @@ var _t = core._t;
 
 tour.register('crm_tour', {
     url: "/web",
-}, [tour.STEPS.MENU_MORE, {
-    trigger: '.o_app[data-menu-xmlid="crm.crm_menu_root"], .oe_menu_toggler[data-menu-xmlid="crm.crm_menu_root"]',
+}, [tour.STEPS.SHOW_APPS_MENU_ITEM, {
+    trigger: '.o_app[data-menu-xmlid="crm.crm_menu_root"], .o_apps_menu_item[data-menu-xmlid="crm.crm_menu_root"]',
```

Comment:
it would have been nice to have a single selector here (i.e. to use o_app instead of o_apps_menu_item)